<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plok Photo Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="mainStackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainStackedThumbnailPage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTreeView" name="fileBrowserTreeView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="rootIsDecorated">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QDeclarativeView" name="thumbnailView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="backgroundBrush">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>128</red>
              <green>128</green>
              <blue>128</blue>
             </color>
            </brush>
           </property>
           <property name="resizeMode">
            <enum>QDeclarativeView::SizeRootObjectToView</enum>
           </property>
           <property name="source">
            <url>
             <string>qrc:/standard/thumbnailView.qml</string>
            </url>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mainStackedPreviewPage">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QDeclarativeView" name="setView">
           <property name="resizeMode">
            <enum>QDeclarativeView::SizeRootObjectToView</enum>
           </property>
           <property name="source">
            <url>
             <string>qrc:/standard/setview.qml</string>
            </url>
           </property>
          </widget>
          <widget class="QDeclarativeView" name="thumbnailNavigator">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
           <property name="interactive">
            <bool>true</bool>
           </property>
           <property name="resizeMode">
            <enum>QDeclarativeView::SizeRootObjectToView</enum>
           </property>
           <property name="source">
            <url>
             <string>qrc:/standard/thumbnailNavigator.qml</string>
            </url>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionUpload_images"/>
    <addaction name="actionCreate_set"/>
    <addaction name="separator"/>
    <addaction name="actionStack"/>
    <addaction name="actionDestack"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImport"/>
   <addaction name="actionLibrary"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionEdit"/>
  </widget>
  <action name="actionUpload_images">
   <property name="text">
    <string>Upload image</string>
   </property>
  </action>
  <action name="actionLibrary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/standard/icons/application-epub+zip.png</normaloff>:/standard/icons/application-epub+zip.png</iconset>
   </property>
   <property name="text">
    <string>Library</string>
   </property>
   <property name="toolTip">
    <string>Library</string>
   </property>
  </action>
  <action name="actionStack">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/standard/icons/22x22/archive-insert.png</normaloff>:/standard/icons/22x22/archive-insert.png</iconset>
   </property>
   <property name="text">
    <string>Stack</string>
   </property>
  </action>
  <action name="actionDestack">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/standard/icons/22x22/archive-remove.png</normaloff>:/standard/icons/22x22/archive-remove.png</iconset>
   </property>
   <property name="text">
    <string>Destack</string>
   </property>
   <property name="toolTip">
    <string>Destack</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/standard/icons/22x22/trash-empty.png</normaloff>:/standard/icons/22x22/trash-empty.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionCreate_set">
   <property name="text">
    <string>Create set</string>
   </property>
  </action>
  <action name="actionDelete_set">
   <property name="text">
    <string>Delete_set</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/standard/icons/22x22/edit-guides.png</normaloff>:/standard/icons/22x22/edit-guides.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/standard/icons/camera-photo.png</normaloff>:/standard/icons/camera-photo.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QDeclarativeView</class>
   <extends>QGraphicsView</extends>
   <header>QtDeclarative/QDeclarativeView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
